### ISSUE TITLE
How to connect to shelley_testnet network from node on version 1.14.2?

### ISSUE THREAD

**Issue by gsakethreddy** [input-output-hk/cardano-graphql#230]
Shelley_testnet network genesis is being connected form node v1.15.1 and not from v1.14.2 after genesis reset. Any plans on upgrading the node to 1.15.1?

**Comment by rhyslbw** [comment]
Yes, there will be a pre-release targeting the Shelley-compatible networks very soon.
